### PR TITLE
Dreamview: implement connection lost detection and consolidate uttera…

### PR DESCRIPTION
**Pull request by vlin17**
…nce logic

### CODE REVIEW

File: modules/dreamview/frontend/src/store/websocket/websocket_realtime.js [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
+import UTTERANCE from "store/utterance";
 import Worker from 'utils/webworker.js';
 
 export default class RosWebSocketEndpoint {
     constructor(serverAddr) {
         this.serverAddr = serverAddr;
         this.websocket = null;
         this.simWorldUpdatePeriodMs = 100;
-        this.simWorldLastUpdateTimestamp = 0;
+        this.simWorldLastUpdateTimestamp = new Date().getTime();
```
Comment (siyangy) [inline modules/dreamview/frontend/src/store/websocket/websocket_realtime.js]: checkMessage relies on the fact that this variable is initially 0. The connection lost alert can follow the same check.

Comment (vlin17) [reply modules/dreamview/frontend/src/store/websocket/websocket_realtime.js]: Good catch. Fixed the logic.

**Comment by siyangy** [comment]
@vlin17 Does this need to be in stable branch as well?

**Comment by vlin17** [comment]
Yes, will cherry-pick to stable branch once merged.

**Pull request closed without merging**